components/scientific/pari/pari.p5m

```diff
@@ -1,19 +1,3 @@
-#
-# This file and its contents are supplied under the terms of the
-# Common Development and Distribution License ("CDDL"), version 1.0.
-# You may only use this file in accordance with the terms of version
-# 1.0 of the CDDL.
-#
-# A full copy of the text of the CDDL should have accompanied this
-# source.  A copy of the CDDL is also available via the Internet at
-# http://www.illumos.org/license/CDDL.
-#
-
-#
-# Copyright (c) 2021,2022 David Stes
-# Copyright (c) 2023 Andreas Wacknitz
-# Copyright (c) 2024 David Stes
-#
 
```

Comment:
The license information here is for exactly this manifest file and has nothing to do with the license of pari itself.
As its contents is generated and no additional contents it is ok for me but I suspect that the PARI team is misled here.

Reply:
I am OK with the request and I hope that the PARI team continues to support OpenIndiana.   They can also reasonably ask for some options to be set in the Makefile or the manifest.